src/jsc/bindings/node/JSNodeHTTPServerSocket.cpp

```
@@ -915,3 +916,21 @@ JSC::Structure* createNodeHTTPServerSocketStructure(JSC::VM& vm, JSC::JSGlobalOb
+
+extern "C" [[ZIG_EXPORT(nothrow)]] int64_t NodeHTTP__getServerSocketFd(JSC::EncodedJSValue value)
+{
+    JSC::JSValue decoded = JSC::JSValue::decode(value);
+    auto* socket = dynamicDowncast<Bun::JSNodeHTTPServerSocket>(decoded);
+    if (!socket || !socket->socket || us_socket_is_closed(socket->socket))
+        return -1;
+    return (int64_t)us_socket_get_fd(socket->socket);
+}
+
+extern "C" [[ZIG_EXPORT(nothrow)]] void NodeHTTP__pauseServerSocket(JSC::EncodedJSValue value)
+{
```

Cppcheck (2.21.0)

[style] The function 'clearOnWritable' is never used.

(unusedFunction)